<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FisheyeCalibrationWidget</class>
 <widget class="QWidget" name="FisheyeCalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="imagePage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="nx::client::desktop::FisheyeCalibrationImageWidget" name="imageWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loadingPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QnProgressWidget" name="loadingWidget" native="true"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::client::desktop::FisheyeCalibrationImageWidget</class>
   <extends>QWidget</extends>
   <header>fisheye_calibration_image_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnProgressWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/common/progress_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
